WordPress/UITestsFoundation/XCUIApplication+SavePassword.swift

```diff
@@ -6,7 +6,7 @@ extension XCUIApplication {
     // This method encapsulates the logic to dimiss the prompt.
     func dismissSavePasswordPrompt() {
         XCTContext.runActivity(named: "Dismiss save password prompt if needed.") { _ in
-            guard buttons["Save Password"].waitForExistence(timeout: 5) else { return  }
+            guard buttons["Save Password"].waitForExistence(timeout: 10) else { return  }
```

Comment:
A few screenshots show the Save Password modal displayed when the test has moved on to the next step, which indicates that the 5-second timeout here isn't enough anymore.

Example:
<img width="423" alt="image" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/17252150/8a738739-3033-4602-bfea-96b3b4013eaa">